lib/cosmos/system/target.rb

```diff
@@ -167,9 +179,9 @@ def get_target_dir(path, name, gem_path)
         dir = File.join(path, name)
       end
       lib_dir = File.join(dir, 'lib')
-      Cosmos.add_to_search_path(lib_dir, false) if File.exist?(lib_dir)
+      Cosmos.add_to_search_path(lib_dir) if File.exist?(lib_dir)
```

Comment:
The project lib folder should win over targets.   There always has to be a way to override things.  In this case, you might want to reuse a target untouched in the config targets folder (or in a gem), but change then something.  You can do that in the project lib folder.